cmd/snap/cmd_auto_import.go

```diff
@@ -153,6 +148,20 @@ func init() {
 	cmd.hidden = true
 }
 
+func isOwned() (bool, error) {
+	users, err := Client().Users()
+	if err != nil {
+		return false, err
+	}
+
+	return len(users) > 0, nil
+}
+
+func autoAddUsers() error {
+	_, err := Client().CreateUsers([]*client.CreateUserOptions{{Known: true}})
```

Comment:
Yes, I fixed this to reuse the existing cmdCreateUsers which will print and avoid code duplication.